src/transport.js

```diff
@@ -35,6 +35,17 @@ module.exports = env => {
           domain: env('MAILGUN_DOMAIN')
         }
       }))
+    case 'smtp':
+      return nodemailer.createTransport({
+        host: env('SMTP_HOST'),
+        port: env('SMTP_PORT'),
+        secure: (env('SMTP_SECURE') == 'true'), // use TLS,
```

Comment:
It could use the strong check `===`

Reply:
The result is the same due to "true" == true is false and "true" === true is false in JS.